pkg/limayaml/defaults.go

```diff
@@ -59,6 +78,13 @@ func FillDefault(y *LimaYAML, filePath string) {
 	if y.Containerd.User == nil {
 		y.Containerd.User = &[]bool{true}[0]
 	}
+	y.Containerd.Archives = defaultContainerdArchives()
```

Comment:
This still needs to be protected by `if len(y.Containerd.Archives) == 0`.

Reply:
fixed 🤦‍♂️